koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -329,7 +329,7 @@ def enabled_tags_model(self):
     def csv_columns(self):
         """Return the required CSV columns if we need to filter them"""
         if self.provider_type == Provider.PROVIDER_AWS:
-            return CSV_REQUIRED_COLUMNS.get(Provider.PROVIDER_AWS)
+            return CSV_REQUIRED_COLUMNS.get(Provider.PROVIDER_AWS) + CSV_REQUIRED_COLUMNS.get("AWS-custom")
```

Comment:
Before, this would return `None` or a tuple. Now it could potentially raise a `TypeError` if either results in `None`.

This should get the key directly, which will raise a more obvious `KeyError` if something is amiss.

```suggestion
            return CSV_REQUIRED_COLUMNS[Provider.PROVIDER_AWS] + CSV_REQUIRED_COLUMNS["AWS-custom"]
```